CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 ## 2.4.0 (Unreleased)
+IMPROVEMENTS:
+
+* Change resource handling to use locking mechanism when resource parallel handling is not supported by vDC. [#255] 
```

Comment:
- "vDC" -> "vCD"
- `[#255]` should be in "GH-" format for the build system to parse

Reply:
Fixed